tools/README.md

```diff
@@ -36,11 +36,20 @@ cargo build --release
 
 ### Launch Script
 
-Run `yarn run build-moonbeam-launch` to install the correct dependency - Installs PureStake moonbeam-launch branch
+Run `yarn run build-moonbeam-launch` to install the correct dependency
 
-Run `yarn run moonbeam-launch` to start a network with `config_moonbeam.json` - Installs PureStake moonbeam-launch branch - Starts a local network with `config_moonbeam.json`
+- Installs PureStake moonbeam-launch branch
 
-Run `yarn run moonbeam-test`, if you want to run a simple test sending transactions to different addresses: - Installs PureStake moonbeam-launch branch - Starts a local network with `config_moonbeam.json` - Runs a simple test sending transactions and testing propagation
+Run `yarn run moonbeam-launch` to start a network with `config_moonbeam.json`
+
+- Installs PureStake moonbeam-launch branch
+- Starts a local network with `config_moonbeam.json`
+
+Run `yarn run moonbeam-test`, if you want to run a simple test sending transactions to different addresses:
+
+- Installs PureStake moonbeam-launch branch
+- Starts a local network with `config_moonbeam.json`
+- Runs a simple test sending transactions and testing propagation
```

Comment:
what happened here? :eyes:

Reply:
Some formatter added a space, not significant in any way this is an `md` file